.github/actions/get-build-info/action.yml

```diff
@@ -0,0 +1,19 @@
+name: "Get build info"
+description: "Get build-related info from GitHub and load it as variables into the runner env"
+runs:
+  using: composite
+  steps:
+    - uses: Alfresco/alfresco-build-tools/.github/actions/get-branch-name@v1.25.0
+    - name: "Get build info"
+      run: |
+        [[ $GITHUB_EVENT_NAME == "pull_request" ]] && IS_PULL_REQUEST="true" || IS_PULL_REQUEST="false"
+        echo "PULL_REQUEST=$IS_PULL_REQUEST" >> "$GITHUB_ENV"
+        echo "BUILD_NUMBER=$GITHUB_RUN_NUMBER" >> "$GITHUB_ENV"
+        echo "ATTEMPT_NUMBER=$GITHUB_RUN_ATTEMPT" >> "$GITHUB_ENV"
+        echo "BUILD_STAGE_NAME=$GITHUB_JOB" >> "$GITHUB_ENV"
```

Comment:
not sure if stage should be used given that is was a travis concept

it's ok to try to be CI agnostic with environment variables naming but in this case can increase confusion, I would keep job name since it's a shared concept also in some other CI platforms (gitlab, circleci)

Reply:
Thanks for the feedback, renamed the `BUILD_STAGE_NAME` variable to `JOB_NAME` 👍